Sprout/Models/Post.h

```diff
@@ -14,7 +14,15 @@ NS_ASSUME_NONNULL_BEGIN
 
 @property (nonatomic, strong) NSNumber *likeCount;
 @property (nonatomic, strong) PFUser *author;
+@property (nonatomic, strong) NSNumber *completedTasks;
+@property (nonatomic, strong) NSNumber *totalTasks;
+@property (nonatomic, strong) NSString *caption;
 
++ (void)createPost:(NSString *)caption
+        withCompleted:(NSNumber *)completedTasks
+         withTotal:(NSNumber *)totalTasks
+        withCompletion:(PFBooleanResultBlock _Nullable)completion;
```

Comment:
If you select these all and hit ctrl-i then xcode will align them along the ":" which is really strange but normal for xcode.